requirements/test.txt

```diff
@@ -11,28 +11,28 @@ attrs==23.1.0
     # via hypothesis
 beautifulsoup4==4.12.2
     # via
-    #   -r test.in
+    #   -r requirements/test.in
```

Comment:
I you run `pip-compile-multi` from within the `requirements` folder you won't get this update noise.